coffee-module/coffee-module-mongodb/pom.xml

```diff
@@ -25,5 +25,14 @@
 			<artifactId>commons-collections</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>org.osgi.service.cdi</artifactId>
```

Comment:
ezek miert kellenek?

Reply:
bukott a javaDoc generálás, mivel a 2.0-ás @ConfigProperty tartalmaz osgi függőséget, amik úgy láttam provided-ek a config pom-jában is

Reply:
https://github.com/eclipse/microprofile-config/issues/716 okozza a hibat, ha javitjak kiszedjuk mi is